crypto/src/main/scala/co/topl/crypto/signing/eddsa/EC.scala

```diff
@@ -53,7 +53,11 @@ trait EC {
   private[signing] val PUBLIC_KEY_SIZE: Int = POINT_BYTES
   private[signing] val SECRET_KEY_SIZE: Int = 32
   private[signing] val SIGNATURE_SIZE: Int = POINT_BYTES + SCALAR_BYTES
-  private[signing] val DOM2_PREFIX: Array[Byte] = "SigEd25519 no Ed25519 collisions".getBytes()
+
+  // "SigEd25519 no Ed25519 collisions" as UTF-8
+  private[signing] val DOM2_PREFIX: Array[Byte] = Array(0x53, 0x69, 0x67, 0x45, 0x64, 0x32, 0x35, 0x35, 0x31, 0x39,
+    0x20, 0x6e, 0x6f, 0x20, 0x45, 0x64, 0x32, 0x35, 0x35, 0x31, 0x39, 0x20, 0x63, 0x6f, 0x6c, 0x6c, 0x69, 0x73, 0x69,
+    0x6f, 0x6e, 0x73)
```

Comment:
If the purpose is to ensure that "SigEd25519 no Ed25519 collisions" encodes to "UTF-8", perhaps we could specify this character set in `getBytes` instead to ensure readability?

```suggestion
  private[signing] val DOM2_PREFIX: Array[Byte] = "SigEd25519 no Ed25519 collisions".getBytes(StandardCharsets.UTF_8)

```

Reply:
I prefer directly setting the value since this is how BouncyCastle does it. But either method will lead to the same result,

Reply:
I discussed these 2 approaches with the team and they had a stronger preference for readability (even at this low-level). 
The suggestion was to pass-in the charset as an argument to `getBytes` and leave a comment warning future devs not to change it and include the explicit bytes (or a link to the bouncy castle implementation) in the comment.

Reply:
Just my two-cents: I also recommend specifying the array's bytes directly. While it won't benefit from "compile time constant optimization" (because it's an array), it is still slightly more performant to skip the `.getBytes` call. But in addition, I suggest leaving `// private[signing] val DOM2_PREFIX: Array[Byte] = "SigEd25519 no Ed25519 collisions".getBytes(StandardCharsets.UTF_8)` as a comment to make it clear where the bytes came from originally.

Reply:
@scasplte2 The team re-discussed this during check-in; you can follow Sean's suggestion above

Reply:
updated with the recommended comment